### PR TITLE
[DRAFT] Implements View-Only Sketch Page

### PR DESCRIPTION
**Pull request by malsf21**
Draft PR, will get updated as more info gets added.

So far, added the implementation for `getSketch(docID)`,  and the router path - will need to implement view (which @emmyc is taking the lead on atm).

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -65,6 +66,11 @@ class App extends React.Component {
     }
   };
 
+  getProgram = async programid => {
+    const { ok, sketch } = await fetch.getSketch(programid);
+    return ok ? true : false;
+  };
+
```
Comment (malsf21) [inline src/components/app.js]: @alyssamw this breaks the CI, not using `sketch`

File: src/components/app.js [hunk 2]
```diff
@@ -65,6 +66,11 @@ class App extends React.Component {
     }
   };
 
+  getProgram = async programid => {
+    const { ok, sketch } = await fetch.getSketch(programid);
+    return ok && sketch ? true : false;
+  };
+
```
Comment (malsf21) [inline src/components/app.js]: Does this pass the sketch information to the routed path? Just noticed that you're not actually returning any sketch information to `MainContainer`.

Comment (alyssamw) [reply src/components/app.js]: For now it's not passed. Should I just get rid of it or might we need the sketch info later?

Comment (malsf21) [reply src/components/app.js]: Right right, I think it'll get passed to the component - that's what I meant by dummy component earlier. Let me see if I can patch together a basic version for this branch.

As a sidenote, means we're probably not going to merge until the feature is done.

**Comment by malsf21** [comment]
Going to merge `master` into this now - it'll be the final branch where we add the actual functionality for `view-only`.

**Comment by malsf21** [comment]
Officially tagged @emmyc into this. Will likely merge our theming PR into this branch, since we'll use that for the view-only view.

**Comment by malsf21** [comment]
Remerged master into this branch. Officially re-tagging @jamieliu386 back into the ticket.

**Comment by malsf21** [comment]
Superseded by #219

**Pull request closed without merging**